app/controllers/tag_controller.rb

```diff
@@ -1,6 +1,5 @@
 class TagController < ApplicationController
   respond_to :html, :xml, :json, :ics
-  before_action :require_user, only: %i(create delete add_parent)
```

Comment:
Hi, actually here I believe we need to remove just the `add_parent` part, but keep the rest of the line!

Reply:
Oh ok, got it!